doc/source/ray-core/walkthrough.rst

```diff
@@ -37,7 +37,7 @@ The first step is to import and initialize Ray:
 
 .. note::
 
-  In recent versions of Ray (>=1.5), ``ray.init()`` is automatically called on the first use of a Ray remote API.
+  If ``ray.init()`` isn't called explicitly, it will implicitly be called with no arguments on the first use of a Ray remote API.
```

Comment:
Suggested edit for simplicity (removes passive voice and future tense as well)

```suggestion
  Unless you explicitly call ``ray.init()``, the first use of a Ray remote API calls ``ray.init()`` with no arguments.
```

Reply:
sounds good, just kept the word "implicitly" in there for clarity